app/src/main/java/ch/epfl/sdp/ui/event/DefaultEventFragment.java

```diff
@@ -46,6 +46,7 @@
     private Sharing mEventSharing;
     private float mZoomLevel = 15;
     private int LAUNCH_CALENDAR = 3;
+    private  int THREE_HOURS = 10800000;
```

Comment:
- Remove extra space between `private` and `int`, and maybe add `final` too.
- Probably better to explain how you get this number. 
For example: ```THREE HOURS = 3 * 60 * 60 * 1000```